packages/http/src/stamp.ts

```diff
@@ -1,38 +1,137 @@
-import * as crypto from "crypto";
+import { webcrypto } from "crypto";
 
 // Specific byte-sequence for curve prime256v1 (DER encoding)
 const PRIVATE_KEY_PREFIX = Buffer.from(
   "308141020100301306072a8648ce3d020106082a8648ce3d030107042730250201010420",
   "hex"
 );
 
-export function stamp(input: {
+export async function stamp(input: {
   content: string;
   publicKey: string;
   privateKey: string;
 }) {
   const { content, publicKey, privateKey } = input;
 
-  const privateKeyBuffer = Buffer.from(privateKey, "hex");
+  const key = await importPrivateKey(privateKey);
+  const signature = await signMessage(key, content);
+
+  return {
+    publicKey: publicKey,
+    scheme: "SIGNATURE_SCHEME_TK_API_P256",
+    signature: signature,
+  };
+}
+
+async function importPrivateKey(
+  privateKeyHex: string
+): Promise<webcrypto.CryptoKey> {
+  const privateKeyBuffer = Buffer.from(privateKeyHex, "hex");
   const privateKeyPkcs8Der = Buffer.concat([
     PRIVATE_KEY_PREFIX,
     privateKeyBuffer,
   ]);
 
-  const privateKeyObject = crypto.createPrivateKey({
-    type: "pkcs8",
-    format: "der",
-    key: privateKeyPkcs8Der,
-  });
+  return await webcrypto.subtle.importKey(
+    "pkcs8",
```

Comment:
`webcrypto.subtle.importKey` supports `raw` as encoding. Can we use this instead of pkcs8? That'd be easier to deal with if it works!

Reply:
😿 

Dang it. Well, thanks for checking!